Source/WebKit/Shared/Cocoa/APIObject.mm

```diff
@@ -437,6 +438,12 @@
     case Type::WebExtensionMessagePort:
         wrapper = [WKWebExtensionMessagePort alloc];
         break;
+
+#if ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)
+    case Type::WebExtensionSidebar:
+        wrapper = [_WKWebExtensionSidebar alloc];
+        break;
+#endif // ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)
 #endif
```

Comment:
i think it would be better to actually add this for the `endif` below
`#endif // ENABLE(WK_WEB_EXTENSIONS)`

the line enwrapping `ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)` are pretty short so i don't really think it's needed